gmso/tests/test_topology.py

```diff
@@ -694,3 +694,36 @@ def test_topology_invalid_scaling_factors(self, typed_methylnitroaniline):
             typed_methylnitroaniline.scaling_factors = (0.5, 1.0)
         with pytest.raises(GMSOError):
             typed_methylnitroaniline.scaling_factors = {"lj_12": 0.0}
+
+    def test_is_typed_check(self, typed_chloroethanol):
+        groups = [
+            "sites",
+            "bonds",
+            "angles",
+            "dihedrals",
+            "impropers",
+            "topology",
+        ]
+        for group in groups:
+            assert typed_chloroethanol.is_fully_typed(group=group)
+
+        with pytest.raises(ValueError):
+            typed_chloroethanol.is_fully_typed(group="foo")
+
+    def get_untyped(self, typed_chloroethanol):
```

Comment:
Lets add one more test that checks for more than 1 untyped objects in a group. Just want to make sure the lists returned by `get_untyped` handle that case properly.